addons/bestja_frontend_fpbz/static/src/css/custom.css

```diff
@@ -1,563 +1,677 @@
-    }
-}
-
-.footer_icons li{
-    display: inline-block;
-    list-style: none;
-    margin-left: 25px;
-    margin-right: 25px;
-    text-align: right;
-    margin-bottom: 20px;
-}
-
-@media screen and (max-width: 991px) {
-    .footer_icons li{
-        text-align: center;
-        float: none;
-    }
-}
-
-.footer_icons li a.facebook{
-    background-image: url('/bestja_frontend_fpbz/static/src/img/content/facebook-02-white.svg');
-    display: inline-block;
-    background-repeat: no-repeat;
-    width: 45px;
-    height: 45px;
-    vertical-align: middle;
-    margin-left: 7px;
-}
-
-.footer_icons li a.twitter{
-    background-image: url('/bestja_frontend_fpbz/static/src/img/content/twitter-02-white.svg');
-    display: inline-block;
-    background-repeat: no-repeat;
-    width: 45px;
-    height: 45px;
-    vertical-align: middle;
-}
-
-.footer_icons li a.youtube{
-    background-image: url('/bestja_frontend_fpbz/static/src/img/content/youtube-02-white.svg');
-    display: inline-block;
-    background-repeat: no-repeat;
-    width: 45px;
-    height: 45px;
-    vertical-align: middle;
-}
-
-.footer_div{
-    background-color: #F39100;
-    min-height: 100px;
-    padding-top: 25px;
-}
-
-.footer_div a{
-    color: #FFFFFF;
-    text-decoration: none;
-}
-
-.footer_div h4 a{
-    margin-left: 20px;
-    margin-right: 20px;
-}
-
-.copyrights p{
-    font-size: 14pt;
-    color: #FFFFFF;
-    text-align: right;
-}
-
-@media screen and (max-width: 991px) {
-    .copyrights p{
-        text-align: center;
-    }
-}
-
-.white_footer_text{
-    color: white;
-}
-
-.footer_div span{
-    margin-left: 12px;
-    margin-right: 12px;
-}
-
-.footer_contact{
-    padding: 0;
-}
-
-.footer_contact li{
-    display: inline-block;
-    list-style: none;
-    margin-bottom: 20px;
-}
-
-.footer_contact li, .footer_contact li a{
-    color: #FFFFFF;
-    font-size: 14pt;
-    margin-left: 10px;
-}
-
-@media screen and (max-width: 991px) {
-    .copyrights{
-        text-align: center;
-    }
-
-    .copyrights p{
-        padding-right: 0;
-    }
-}
-/* End of footer */
-
-
-
-/* Navbar font modification*/
-
-.navbar-static-top{
-    font-size: 24pt;
-}
-
-.dropdown-menu > li > a{
-    font-size: 14pt;
-    font-weight: 600;
-}
-
-/* End of Navbar font modification*/
\ No newline at end of file
+@charset "utf-8";
+@import url(http://fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,600italic,700italic,700,300,600,400&subset=latin,latin-ext);
+
+/* Common css */
+#wrap, section{
+    font-family: 'Open Sans', sans-serif;
+    font-size: 20px;
+}
+
+.open_sans_bold{
+    font-family: 'Open Sans', sans-serif;
+    font-weight: 700;
+}
+
+section .container-fluid{
+    margin-left: 80px;
+    margin-right: 80px;
+}
+
+@media screen and (max-width: 991px) {
+    section .container-fluid{
+        margin-left: 0;
+        margin-right: 0;
+    }
+}
+
+section .top-section-container{
+    margin-left: 0;
+    margin-right: 0;
+}
+
+h1, .h1{

```

Comment:
That is how Boostrap defines its css for headers. Should I remove those classes?